src/components/overlays/AddIDP/index.tsx

```diff
@@ -182,33 +245,91 @@ export const AddIdp = () => {
       await updateIdp(idpUpdateData).unwrap()
       dispatch(show(OVERLAYS.UPDATE_IDP, idp.identityProviderId))
       success(t('add.success'))
+      setLoading(false)
     } catch (err) {
       error(t('add.error'), t('state.error'), err as object)
+      setLoading(false)
     }
```

Comment:
when we end both blocks try and catch with the same command setLoading(false) it's better to put it only once after the closing catch

Reply:
Noted.